src/open_inwoner/accounts/models.py

```diff
@@ -119,6 +119,11 @@ class User(AbstractBaseUser, PermissionsMixin):
     # TODO fix rsin & bsn to not be both null AND blank (!)
     rsin = models.CharField(verbose_name=_("Rsin"), max_length=9, null=True, blank=True)
     bsn = NLBSNField(verbose_name=_("Bsn"), null=True, blank=True)
+    # KvK number is the same as BSN format-wise, thus we use the same class
+    kvk = NLBSNField(verbose_name=_("KvK"), null=True, blank=True)
+    trade_name = models.CharField(
```

Comment:
Ik zou hier 'company_name' van maken (en eventuele additionele velden ook een 'company_' prefix geven). Hoewel het de juiste vertaling is van handelsnaam, 'trade_name' voelt hier in niet echt passend.

Daarnaast kun je BSN-validatie (9 cijfers, elf-proef van toepassing) niet toepassen op een KvK nummer (8 cijfers, geen elf-proef). Neem maar even een blik op VIPS:

https://bitbucket.org/maykinmedia/belastingdienst-gegevensstromen/src/456bd4d51616677c1830347993891f7a8643b4fa/src/vips/accounts/models/company.py#lines-148